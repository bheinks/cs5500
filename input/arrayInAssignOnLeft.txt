(* Cannot have entire array on lhs of assignment *)

program p;

var A : array [1..5] of integer;

begin
  A := 5;
  write('!')
end.

