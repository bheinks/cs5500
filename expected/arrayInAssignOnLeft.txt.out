Line 8: Array variable must be indexed
